Client/Frontend/Strings.swift

```diff
@@ -357,8 +357,8 @@ extension String {
             public static let Header = MZLocalizedString(
                 key: "CreditCard.RememberCard.Header.v115",
```

Comment:
We should change the key if the value and comments are changed?

Reply:
No we should not, given that the string is for 115. Just fixing stuff that was missed in the pass.

Reply:
Ah ok it wasn't exported yet 👍